src/components/moderation/Hider.tsx

```diff
@@ -27,13 +27,17 @@ export const useHider = () => React.useContext(Context)
 
 export function Outer({
   modui,
+  override,
   children,
 }: React.PropsWithChildren<{
+  override?: boolean
```

Comment:
one additional prop that lets us supply some override, like if the hidden content belongs to the current session's account.